app.rb

```diff
@@ -0,0 +1,148 @@
+require_relative 'person'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+def all_books(books)
+  puts 'List of books:'
+  puts
+  books.each_with_index do |book, index|
+    puts "#{index + 1}. Title: #{book.title}, Author: #{book.author}"
+    puts
+  end
+end
+
+def all_people(people)
+  puts 'List of people:'
+  puts
+  people.each_with_index do |person, index|
+    puts "#{index + 1}. Name: #{person.name}, ID: #{person.id}"
+    puts
+  end
+end
```

Comment:
- [ ] Kindly prefix each person with his role and the age the user should know that tha person is either a student or a teacher just like in the video exaple:

your list | the video description 
----|----
<img width="359" alt="image" src="https://github.com/Santosh-Konappanavar/school_library/assets/65068771/747c3088-40c9-4bfe-b9e1-8aadb9850662"> | <img width="359" alt="image" src="https://github.com/Santosh-Konappanavar/school_library/assets/65068771/69ab9586-3de5-48df-b5b4-3a4a6a74edf9">